src/Datagrid/Datagrid.php

```diff
@@ -291,4 +291,27 @@ public function getForm()
 
         return $this->form;
     }
+
+    /**
+     * Returns the default filter value if the user has not set it.
+     *
+     * @return array|null
+     */
+    protected function resolveFilterValue(?array $data, string $name, FilterInterface $filter)
```

Comment:
Could we reduce visibility?
```suggestion
     */
    protected function resolveFilterValue(?array $data, string $name, FilterInterface $filter): ?array
```